.github/workflows/ci.yml

```diff
@@ -11,21 +11,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [8, 11]
+        java: [11]
```

Comment:
Should we add Java 17 as well?

Reply:
I didn't add to this repo because it almost has no Java code (only a single simple class), can add if you think it's a must.